errors.go

```diff
@@ -106,6 +106,23 @@ func New(message string) error {
 	}
 }
 
+// ExistStack return an error already with stack
+// ExistStack will check all parent error
+func ExistStack(err error) bool {
+	type stackTracer interface {
+		StackTrace() StackTrace
+	}
+	if _, ok := err.(stackTracer); ok {
+		return true
+	}
+	type causer interface {
+		Cause() error
+	}
```

Comment:
Just checking `Cause() error` ignores the possibility of an `Unwrap() error` which is kind of the preferred way of unwrapping errors now. So, this would only work with errors from this package, and those designed specifically for pre-unwrap compatibility with this package, which is no longer a strong argument since go1.13.